comp/core/tagger/global.go

```diff
@@ -44,8 +44,25 @@ func GetEntity(entityID types.EntityID) (*types.Entity, error) {
 	return globalTagger.GetEntity(entityID)
 }
 
+// LegacyTag is an interface function that queries taggerclient singleton
+// If possible, avoid using this function, and use the Tag interface function instead.
+// This function exists in order not to break backward compatibility with rtloader and python
+// integrations using the tagger
+func LegacyTag(entity string, cardinality types.TagCardinality) ([]string, error) {
```

Comment:
I see your point, but the thing here is that we need to push developers away from using this method because it includes an extra step of string parsing, which we can avoid by calling `Tag` method of the tagger. 

This is why I think it is good to include something in the function name indicating that calling it should be avoided except in very specific cases. 

How about: 

`LegacyTagByString` ?